packages/trader/src/Modules/MT5/Containers/mt5-compare-accounts-modal.jsx

```diff
@@ -46,140 +46,154 @@ const MT5AttributeDescriber = ({ name, tooltip, counter }) => {
     );
 };
 
-const filterAvailableAccounts = (landing_companies, table) => {
+const filterAvailableAccounts = (landing_companies, table, is_eu) => {
     return table.map(({ attribute, synthetic, financial_stp, financial }) => {
+        if (is_eu) {
+            return {
+                attribute,
+                ...(landing_companies?.mt_gaming_company?.financial ? { synthetic } : {}),
```

Comment:
how about for EU countries that we don't allow account opening in and will have empty values?

Reply:
or is this based on residence and not IP address?

Reply:
I believe it's based on `residence`.